test/__snapshots__/PublishCommand.js.snap

```diff
@@ -1,13 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`[independent --canary --npm-tag=next --yes --exact] npm dist-tag add 1`] = `
```

Comment:
Think I updated everything in [#4ed5845](https://github.com/lerna/lerna/pull/777/commits/4ed58452bdd92813a21ba31ae1a6d9526f3d9525), dunno why github isn't detecting it.